GOVERNANCE.md

```diff
@@ -0,0 +1,133 @@
+# Release Working Group
+
+The Node.js Release Working Group (WG) maintains oversight
+over the Node.js Release, Long Term Support (LTS) and
+Canary in the Gold Mine (CitGM) teams. It manages the release
+and long term support schedule and policies for all Node.js releases.
+
+The WG has final authority over Releases including:
+
+* Release process and tools.
+* Content for all releases.
+* Schedule for all releases.
+* Contribution policy for the Release repository.
+* Conduct guidelines for the Working group.
+
+For the current list of WG members, see the project README.md.
+
+## Collaborators
+
+The Release GitHub repository is maintained by the WG (all WG
+members are Collaborators for the Release repository) and additional
+Collaborators who are added by the WG on an ongoing basis.
+
+Individuals making significant and valuable contributions are made
+Collaborators and given commit-access to the project. These individuals
+are identified by the WG and their addition as Collaborators is discussed
+during the weekly WG meeting.
+
+**Note**: If you make a significant contribution and are not considered for
+commit access, log an issue or contact a WG member directly and it will
+be brought up in the next WG meeting.
+
+Modifications of the contents of the Release repository are made 
+on a collaborative basis. Anybody with a GitHub account may propose a
+modification via pull request and it will be considered by the project
+Collaborators. All pull requests must be reviewed and accepted by a
+Collaborator with sufficient expertise who is able to take full responsibility
+for the change. In the case of pull requests proposed by an existing
+Collaborator, an additional Collaborator is required for sign-off. Consensus
+should be sought if additional Collaborators participate and there is
+disagreement around a particular modification. See Consensus Seeking
+Process below for further detail on the consensus model used for governance.
+
+Collaborators may opt to elevate significant or controversial modifications,
+or modifications that have not found consensus to the WG for discussion by
+assigning the WG-agenda tag to a pull request or issue. The WG should serve
+as the final arbiter where required.
+
+## WG Membership
+
+WG seats are not time-limited. There is no fixed size of the WG.
+
+There is no specific set of requirements or qualifications
+for WG membership beyond these rules.
+
+The WG may add additional members to the WG by unanimous consensus.
+
+A WG member may be removed from the WG by voluntary resignation,
+or by unanimous consensus of all other WG members. If a member is
```

Comment:
Do we define unanimous consensus to mean "lots of people agreed, and no-one disagreed within a certain timeframe"? Because otherwise if (for example) two people in the WG have moved on, then we need Person 1 to agree to remove Person 2 (which might never happen). I feel this is what we do in practice, but IDK if it's defined anywhere.